lib/network/net_test.go

```diff
@@ -80,37 +91,53 @@ func TestSecureMultiClose(t *testing.T) {
 
 	h1 := NewSecureTcpHost(priv1, entity1)
 	h2 := NewSecureTcpHost(priv2, entity2)
+	done := make(chan bool)
 	go func() {
 		err := h1.Listen(fn)
 		if err != nil {
 			t.Fatal("Listening failed for h1:", err)
 		}
+		close(done)
 	}()
-	h2.Open(entity1)
-	err := h1.Close()
+	time.Sleep(time.Second)
```

Comment:
Hmm, in fact it should not be necessary anymore. I'll check it out tomorrow.

And Travis fails again - in a new, unknown place...